modules/RTC/RTC.js

```diff
@@ -102,7 +102,8 @@ function _newCreateLocalTracks(mediaStreamMetaData = []) {
             sourceType,
             stream,
             track,
-            videoType: videoType || null
+            videoType: videoType || null,
+            effects: options.effects
```

Comment:
Isn't that going to add the effect property even on audio tracks? 

Can't we make `obtainAudioAndVideoPermissions` to include it in `tracksInfo` array? This way we wouldn't need this parameter and the effect property will be assigned only to the correct `JitsiLocalTrack`.

Reply:
I thought that maybe in the future we need an effect/processing of audio stream also. And i can give and example if we discuss offline. We can move it there for audio/video media but then i think i have to adapt JitsiLocalTrack also.

Reply:
Also the effects option is used from outside lib-jitsi-meet using JitsiMeetJS.js createLocalTracks otherwise it is empty array

Reply:
From JSDoc of JitsiMeetJS.js createLocalTracks 
* @param options Object with properties / settings specifying the tracks
* which should be created. should be created or some additional
* configurations about resolution for example.

Reply:
Moved effects to trackInfo